packages/components/src/menu-item/style.scss

```diff
@@ -51,9 +51,17 @@
 }
 
 .components-menu-item__shortcut {
+	align-self: center;
 	opacity: 0.5;
 	margin-right: 0;
 	margin-left: auto;
-	align-self: center;
 	padding-left: 8px;
+
+	// Hide the keyboard shortcuts on mobile, because they aren't super-useful
+	// for most mobile users and it frees up screen space for any item
+	// with a long description.
+	display: none;
```

Comment:
Probably fine for now, but I wonder if this should be hidden entirely in the generic MenuItem component or something specific to the Global menu item. 

It can be used in the Block Settings Menu and elsewhere since it's a generic component.